CRONAPP-3853 - Componente de LGPD

alissonsilva-cronapp opened the pull request:
https://cronapp.atlassian.net/browse/CRONAPP-3853

### Problema
### Melhoria
Componente LGPD
### Solução
Desenvolvido Componente Inicial para atender LGPD

alissonsilva-cronapp committed:
CRONAPP-3853 - Componente de LGPD

https://cronapp.atlassian.net/browse/CRONAPP-3853

### Problema
### Melhoria
Componente LGPD
### Solução
Desenvolvido Componente Inicial para atender LGPD

roneliocronapp commented on css/privacy/privacy.css:
```diff
@@ -0,0 +1,59 @@
+/*
+* MATERIAL -> PRIVACY
+*
+* Todo CSS destinado às Tela do LGPD
```
O comentário precisa de um pequeno ajuste.

alissonsilva-cronapp replied:
Ajustado

roneliocronapp reviewed (changes requested):
Precisa ajustar o comentário do CSS de privacidade

alissonsilva-cronapp committed:
CRONAPP-3853 - Componente de LGPD

https://cronapp.atlassian.net/browse/CRONAPP-3853

### Problema
### Melhoria
Componente LGPD
### Solução
Desenvolvido Componente Inicial para atender LGPD

alissonsilva-cronapp committed:
CRONAPP-3853 - Componente de LGPD

https://cronapp.atlassian.net/browse/CRONAPP-3853

### Problema
### Melhoria
Componente LGPD
### Solução
Desenvolvido Componente Inicial para atender LGPD
